### ISSUE TITLE
t1-linear pipeline CAPS name issue

### ISSUE THREAD

**Issue by ravih18** [aramis-lab/clinica#174]
**Describe the bug**
CAPS output of `t1-linear` pipeline have error in file name. The key "T1w" appears twice in the name of the files: once after the subject and session value and once at the end of the file name. 

**To Reproduce**
1. Run t1-linear pipeline
2. Go one the CAPS directory 
3. Check the t1-linear outputs

**Expected behavior**
The name of `t1-linear` pipeline outputs are:
- <source_file>_**T1w**_space-MNI152NLin2009cSym_res-1x1x1_affine.mat  
- <source_file>_**T1w**_space-MNI152NLin2009cSym_res-1x1x1_T1w.nii.gz
instead of:
- <source_file>_space-MNI152NLin2009cSym_res-1x1x1_affine.mat
- <source_file>_space-MNI152NLin2009cSym_res-1x1x1_T1w.nii.gz
as specified in the documentation.

**Desktop (please complete the following information):**
 - OS: Max OS and Linux Ubuntu
 - Version Clinica 3.8

**Comment by mdiazmel** [comment]
Hi @ravih18 !

In fact, this is the normal behavior of the pipeline. The double "T1w" key in the filename assures the compliance with BIDS specification. Maybe @alexandreroutier can provide a more detailed explanation...

**Comment by alexandreroutier** [comment]
That is correct.

BIDS files look like:
```
sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>][_rec-<label>][_run-<index>][_part-<label>]_<suffix>.nii[.gz]
```
so `<source_file>` should be read:
```
```
sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>][_rec-<label>][_run-<index>][_part-<label>]_<suffix>
```
e.g. `<source_file>` of `sub-CLNC01_ses-M00_acq-FDG_run-01_PET.nii.gz` will be `sub-CLNC01_ses-M00_acq-FDG_run-01_PET.nii.gz`

We were aware that `_PET` would be redundant in some cases e.g. registered data in your pipeline. However, this was the simplest solution at that time to trace input data.  In the meantime, BIDS-Derivative now uses JSON files in its latest version: https://bids-specification.readthedocs.io/en/stable/05-derivatives/02-common-data-types.html.

To sum up, some clarification needs to be added to the Wiki.

Alex
